python_modules/automation/automation/docker/images/buildkite-test/Dockerfile

```diff
@@ -129,12 +129,7 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | \
     docker-ce docker-ce-cli containerd.io docker-compose-plugin \
     # Install docker-compose
     && curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose \
-    && chmod +x /usr/local/bin/docker-compose \
```

Comment:
@neilfulwiler I vaguely remember you looking into doing something similar but closing it? Wanted to call your attention to this.